src/plugins/tabs/components/tabs.vue

```diff
@@ -171,6 +205,13 @@ export default {
             this.currentTab = tab;
             this.addTabToLoaded(tab);
         },
+        shouldLoadTab(tab) {
+            const isLoadedAndCurrent = this.isTabLoaded(tab) && this.isCurrent(tab);
```

Comment:
For now, can we at least cache the result of the calls to `isTabLoaded()` and `isCurrent()`?